examples/functions/functions.bal

```diff
@@ -1,15 +1,19 @@
 import ballerina/io;
 
-// This function takes a `string` argument, but does not have a return value.
+// This function takes in a `string` argument, but does not have a return
+// value.
 function printValue(string value) {
     io:println(value);
 }
 
-// This function takes two `int` values and returns their sum as an integer.
+// This function takes in two `integer` values as arguments and returns their
```

Comment:
`int` is fine here right? If using "integer", then let's get rid of the back ticks.